teensy4/usb_audio.cpp

```diff
@@ -94,7 +92,7 @@ void usb_audio_configure(void)
 	printf("usb_audio_configure\n");
 	usb_audio_underrun_count = 0;
 	usb_audio_overrun_count = 0;
-	feedback_accumulator = 739875226; // 44.1 * 2^24
+	feedback_accumulator = (AUDIO_SAMPLE_RATE_EXACT / 1000.0f) * 0x1000000; // samples/millisecond * 2^24
```

Comment:
To keep it consistent with the previous hardcoded number, should probably round up or at least add 1:

```suggestion
	feedback_accumulator = (AUDIO_SAMPLE_RATE_EXACT / 1000.0f) * 0x1000000 + 1; // samples/millisecond * 2^24
```

Reply:
Hmmm ... good point, but as it's multiplied by 2^24 an "error" of one count isn't going to matter much, and presumably it'll promptly get corrected as it's trying to match the PC's demand rate to the Teensy clock. Or something. I'm out of my depth here!

I have checked, and the cast to integer happens late enough that the 0.1 doesn't get lost. There is (maybe) a bit of loss of precision, I get 739875200 printed out, but not 100% sure the adjustments to the value haven't started by then.

Reply:
> Hmmm ... good point, but as it's multiplied by 2^24 an "error" of one count isn't going to matter much, and presumably it'll promptly get corrected as it's trying to match the PC's demand rate to the Teensy clock. Or something. I'm out of my depth here!

Agreed, just wanted to at least note the difference in case there was some math or concept that depended on that number :).

> I have checked, and the cast to integer happens late enough that the 0.1 doesn't get lost. There is (maybe) a bit of loss of precision, I get 739875200 printed out, but not 100% sure the adjustments to the value haven't started by then.

Sorry, I'm not sure I understand what the 0.1 you're referring to is. You get the 739875200 after the code has been running and the accumulator's initial value has been changed I take it?

<img width="923" alt="Screenshot 2022-11-28 at 16 02 33" src="https://user-images.githubusercontent.com/373823/204390323-b97ff6db-a23e-4884-acc5-b9a71a261926.png">

Reply:
I'm assuming so ... 44100 / 1000 = 44.1 ... if the integer cast happen after this calculation, we'd get 44 * 2^24 and the fractional part, 0.1, would be "lost", as I put it. Actually, I think the calculation is done as float, not double, so it's actually probably a loss of precision. Still not really important!